### PR TITLE
bugfix/snowflake-quoted-identifiers

### PR DESCRIPTION
**Pull request by fivetran-joemarkiewicz**
**Are you a current Fivetran customer?** 

Fivetran created PR

**What change(s) does this PR introduce?** 

**Did you update the CHANGELOG?** 

- [X] Yes

**Does this PR introduce a breaking change?**

- [ ] Yes (please provide breaking change details below.)
- [X] No  (please provide an explanation as to how the change is non-breaking below.)

**Did you update the dbt_project.yml files with the version upgrade (please leverage standard semantic versioning)? (In both your main project and integration_tests)** 

- [X] Yes

**Is this PR in response to a previously created Bug or Feature Request**

- [X] Yes, Issue/Feature #68 
- [ ] No 

**How did you test the PR changes?** 

 

- [X] CircleCi  
- [X] Local (please provide additional testing details below)

**Select which warehouse(s) were used to test the PR**
 

- [X] BigQuery
- [X] Redshift
- [X] Snowflake
- [X] Postgres
- [ ] Databricks
- [ ] Other (provide details below)

**Provide an emoji that best describes your current mood**
 
🐺 
**Feedback**

We are so excited you decided to contribute to the Fivetran community dbt package! We continue to work to improve the packages and would greatly appreciate your [feedback](https://www.surveymonkey.com/r/DQ7K7WW) on our existing dbt packages or what you'd like to see next.

### CODE REVIEW

File: macros/sfdc_formula_view.sql [hunk 1]
```diff
@@ -1,4 +1,4 @@
-{%- macro sfdc_formula_view(source_table, source_name='salesforce', reserved_table_name=source_table, fields_to_include=none, full_statement_version=false, materialization='view') -%}
+{%- macro sfdc_formula_view(source_table, source_name='salesforce', reserved_table_name=source_table, fields_to_include=none, full_statement_version=false, materialization='view', using_quoted_identifiers=false) -%}
```
Comment (fivetran-avinash) [inline macros/sfdc_formula_view.sql:1]: More my own curiosity: Checking the readme, these are arguments that the customer will modify depending on their own warehouse/schema variables and fields?

File: macros/sfdc_formula_view.sql [hunk 2]
```diff
@@ -14,9 +14,14 @@
 */
 
 {% if full_statement_version %}
+{% if using_quoted_identifiers %}
+{%- set table_results = dbt_utils.get_column_values(table=source(source_name, 'fivetran_formula_model'), column='"model"', where="\"object\" = '" ~ source_table ~ "'") -%}
```
Comment (fivetran-avinash) [inline macros/sfdc_formula_view.sql:18]: so the models in Snowflake are processed as double-quoted in the `fivetran_formula_model` seed file?
not quite sure I understand the where condition--so the object is also being double-quoted, and it's referencing the object fields in the seed file, and those objects have to be source_tables, but they're being double-quoted in Snowflake, so additional double-quotes are needed? what are the `\` for?

File: integration_tests/models/src_fivetran_formula.yml [hunk 3]
```diff
@@ -2,7 +2,7 @@ version: 2
 
 sources:
   - name: salesforce #This should always be salesforce
-    schema: 'salesforce_formula_integration_tests' #Modify this to be where your Salesforce data resides
+    schema: "{{ var('salesforce_schema', 'salesforce') }}" #Modify this to be where your Salesforce data resides
```
Comment (fivetran-avinash) [inline integration_tests/models/src_fivetran_formula.yml:5]: I understand the `salesforce_schema` part, but why is `salesforce` needed?

**Review by fivetran-avinash — COMMENTED** [review]
@fivetran-joemarkiewicz Review completed! Mostly qualifying questions, I'm sure once you answer I'll be happy to approve.

File: macros/sfdc_formula_view.sql (hunk 1 above)
Comment (fivetran-joemarkiewicz) [reply macros/sfdc_formula_view.sql:1]: Yeah exactly! And a few other non warehouse/schema specific items such as what type of version of the macro they want to use.

**Review by fivetran-avinash — APPROVED** [review]
@fivetran-joemarkiewicz Clarified questions answered!

**Pull request merged**